### ISSUE TITLE
how do we get a representative sample of the eth community?

### ISSUE THREAD

**Issue by owocki** [gitcoinco/ERC-1789#11]
how do we get a representative sample of the eth community?

https://twitter.com/trent_vanepps/status/1103724486405300224

also remember this is the active ETH twitter community, only a small slice and not representative

we could maybe use https://tennagraph.com/

**Comment by lrettig** [comment]
I don't know if it's possible. We could expend an enormous amount of time and resources on this and still not feel that it's enough. The best tools we have the the existing community engagement tools - twitter, reddit, hosting AMAs and open calls for participation, ethereum-magicians.org, and in-person meetups and conferences. I think Tennagraph has potential too.

**Comment by GregTheGreek** [comment]
I agree, we will have to just continually pump it through all sorts of medias, and drive discussion and "vote" to one platform

**Comment by pet3r-pan** [comment]
We probably need to accept that we cannot cover this 100% from day one and even perhaps ever. But doesn't mean we can't improve it.

However, one way of gradually improving this is to periodically identify a segment area of the Ethereum community that can gain from representation then develop/source that individual as a representative. For example:

- **Q1:** We are starting off with representatives from major orgs eg. EF, Consensys, Status, Church of Consensus etc.


- **Q2:** We onboard identified community alliance representitives from the community eg. an DAO community representative of DAOstack, Aragon, DAOincubator etc. + then maybe a representitive of the dapp community eg. an alliance formed by the (State of the Dapps + DappRadar + dapp.com + ...)


- **Q3:** Identify more communities to onboard, rinse repeat. 


**This will require:** 

- Recruitment of community alliance leaders
- Fostering of community alliances when certain areas of the community remains uncoordinated
- Protocols for governing participating members

This can be achieved steadily done but will need to be done with care and will need dedicated energy. It is probably not the most resource conservative approach however, it can be done one way or another through brute force of diplomacy/people wrangling.

**Comment by krisjpi** [comment]
Hey folks, bumping this up again as I think it's good to revisit where we're at in terms of research into community samples, demographics etc.

Have there been any recent major publications or projects in the space that anyone knows of?

I have put a related proposal forward to Marketing DAO, but kinda let it die moving into the second round as I got too busy leading into ETHDenver to finish off the proposal. Will likely push it forward again when I have the bandwidth again to manage it.

In any case, it was a proposal to create a repo of existing community research for easy and historical reference for marketers, researchers, and beyond. The concept would be to fund the initial gathering of the materials, and then run an open bounty for additional contributions to the repo that have emerged or are simply missing. Would love to get a start on this work regardless of its funding status through Marketing DAO, and any follow-on funding can help the project later.

Examples of existing data includes:
Eva Beylin’s Governance Survey (https://medium.com/coinmonks/ethereum-governance-survey-results-c67c11695f2a)
EIP0 responses (https://www.figma.com/file/I0r0qsykpH459ti94LrVql3q/EIP0-Responses?node-id=0%3A1)
Coindesk’s long running State of Blockchain surveys (https://www.coindesk.com/research-archive)
